src/sql-adapter/edfi_sql_adapter/sql_adapter.py

```diff
@@ -137,15 +137,13 @@ def get_int(self, statement: str) -> int:
         Integer result count.
         """
 
-        def __callback(session: sa_Session) -> int:
-            return session.execute(statement).scalar()
+        with self.engine.connect() as connection:
+            result = connection.execute(statement).scalar()
```

Comment:
There's no reason for `get_int` to be wrapped in a transaction. And the error handling in `execute_transaction` was logging an exception that I did not want to be logged (when running the migrations for the first time, throws an exception because the migration table does not yet exist).